LayoutExample.cs

```diff
@@ -0,0 +1,126 @@
+// LayoutExample.cs
```

Comment:
What's this file for? Can we leave an inline comment to define its purpose?

Reply:
Sure, it's just an example of the file header and the file layout order.

Reply:
Resolved in 55c9cc8

Reply:
I would just add something like "all files in this project must follow this template" and remove the copyright header from this example too. Thanks for adding this!